packages/relay-compiler/core/RelayFindGraphQLTags.js

```diff
@@ -23,14 +23,14 @@ import type {
   GraphQLTagFinder,
 } from '../language/RelayLanguagePluginInterface';
 
-const cache = new RelayCompilerCache('RelayFindGraphQLTags', 'v1');
+const cache = new RelayCompilerCache('RelayFindGraphQLTags', 'v2');
```

Comment:
Have to bust the cache, as we should now cache the entire `tag`, instead of only the `tag.template`.

Reply:
@kassens This work is distinct from #2784 imo.